pkg/testutil/testdir_test.go

```diff
@@ -124,21 +124,21 @@ func TestApplyDir_Symlinks(t *testing.T) {
 		{"sf1", filepath.Join("d1", "f")},
 	}
 	for _, fromTo := range symlinkTargets {
-		symlink, expected_target := fromTo[0], fromTo[1]
+		symlink, expectedTarget := fromTo[0], fromTo[1]
```

Comment:
Follow the naming convention elsewhere and call this `wantTarget`.

Reply:
Done, but I think consistently using `exp` (short for "expected") and `act` (short for "actual") unit test var name prefixes is preferable. Especially since, in this case, there is no existing use of `wantTarget` in the source code. 😸

Reply:
The `want` and `got` prefixes are used consistently in all the other test files.